test/js/base/config.js

```diff
@@ -64,4 +64,97 @@ describe("Config", function() {
             assert.strictEqual(instance.hasInitialsRadius(), true);
         });
     });
+
+    describe("#initialsConfig()", function() {
+        it("should be able to retrieve the initials config with the applied profiles", () => {

```

Comment:
Would be nice if we had a test to compare the result of local RIPE SDK `initialsConfig()` against remote RIPE Core `getInitialsConfigP()`. I think considering the risk and all the discussion it's important to add some confidence that the local method is in sync with the remote method.